.github/workflows/test.yml

```diff
@@ -6,30 +6,54 @@ on:
   pull_request:
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
-  unix-test:
+  test:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]
+        os:
+          - ubuntu-latest
+          # TODO: Test other OSs!
+          #- macos-latest
+          #- windows-latest
         emacs-version:
           - 27.2
-          - 28.1
-          - 29.3
+          - 28.2
           - 29.4
+        experimental: [false]
+        include:
+        - os: ubuntu-latest
+          emacs-version: snapshot
```

Comment:
Probably this can be removed ? It is intentional that I don't want to test on snapshot. I have seen occasional failures occurring just on snapshot which marks the entire CI as red IIRC.

Reply:
Got it! Removed!